restcomm/restcomm.sms/src/main/java/org/restcomm/connect/sms/smpp/SmppMessageHandler.java

```diff
@@ -361,7 +361,9 @@ public void outbound(SmppOutboundMessageEntity request) throws SmppInvalidArgume
             textBytes = CharsetUtil.encode(request.getSmppContent(), CharsetUtil.CHARSET_UCS_2);
         } else {
             submit0.setDataCoding(DataCoding.DATA_CODING_GSM7);
-            textBytes = CharsetUtil.encode(request.getSmppContent(), request.getSmppEncoding());
+
+            //FIXME: workaround for https://telestax.atlassian.net/browse/RESTCOMM-1856
+            textBytes = CharsetUtil.encode(request.getSmppContent(), CharsetUtil.CHARSET_ISO_8859_1);
```

Comment:
this is what you mentioned is specific to Provider/nexmo,right?

Reply:
yup